static/js/common/upload-addon.js

```diff
@@ -261,13 +262,99 @@
             });
 
 
+
+
+
```

Comment:
That's quite a lot of newlines... I know this isn't python, but my `flake8` senses are tingling anyway.